.eslintrc.json

```diff
@@ -0,0 +1,316 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "amd": true,
+        "es6": true,
+        "node": true,
+        "mocha": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
```

Comment:
Given that we are extending “eslint:recommended”, it possible to only list the ones we are changing?

Reply:
So it seems we can just remove those `off` lines, but IMO maybe those lines are helpful for future reconsideration (at least we know they exist!). Should they be removed?

Reply:
I don’t mind keeping it as is, but I’m worried that we don’t know which are set by us or why - and the recommended set can change or have new things added. I guess it’s not a big deal as the code base is only small.